docs/Account.md

```diff
@@ -351,7 +351,8 @@ signature: felt*
 
 Returns:
 
-None.
+`TRUE` if a given signature is valid. Otherwise, reverts. In the future it will return `FALSE` if a given signature is invalid (for more info please check [this issue](https://github.com/OpenZeppelin/cairo-contracts/issues/327)).
+
```

Comment:
> I don't think I understand what you're referring to; but if you add a commit suggestion I'll gladly add it :)

Haha sorry, context would have been helpful :) 
Something like this, what do you think?
``````suggestion
```cairo
is_valid: felt
```

> returns `TRUE` if a given signature is valid. Otherwise, reverts. In the future it will return `FALSE` if a given signature is invalid (for more info please check [this issue](https://github.com/OpenZeppelin/cairo-contracts/issues/327)).
``````